rero_ils/static/scss/rero_ils/styles.scss

```diff
@@ -53,6 +53,12 @@ ng\:form {
   position:absolute;
 }
 
+.nopadding {
```

Comment:
See previous comment about spacing in bootstrap 4.

Reply:
Only for boostrap 4